### ISSUE TITLE
xml error when fetching bucket info

### ISSUE THREAD

**Issue by InformatiQ** [exoscale/pithos#119]
is that xml error an expected one?

```
[rhanna@execbox ~]$ s3cmd mb -P s3://user1-pub
localhost:8080
Bucket 's3://user1-pub/' created
[rhanna@execbox ~]$ s3cmd info s3://user1-pub
localhost:8080
localhost:8080
ERROR: Error parsing xml: Malformed error XML returned from remote server..  ErrorXML: <?xml version="1.0" encoding="UTF-8"?><UnknownAction xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Action><Code>get-bucket-requestpayment</Code></Action></UnknownAction>
s3://user1-pub/ (bucket):
   Location:  CH-GV1
   Payer:     none
localhost:8080
   Expiration Rule: none
localhost:8080
localhost:8080
   policy:    none
localhost:8080
   cors:      <?xml version="1.0" encoding="UTF-8"?><CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"></CORSConfiguration>
   ACL:       user1: FULL_CONTROL
   ACL:       *anon*: READ
   URL:       http://user1-pub.s3.execbox.local:8080/
```

**Comment by pyr** [comment]
There is no support for `get-bucket-requestpayment`.

**Comment by pyr** [comment]
I'll add a stub for it in the coming days, thanks for the report.

**Comment by pyr** [comment]
#126 and #127 address the s3cmd output errors, thanks!

**Comment by brutasse** [comment]
Fixed in #127
